sphinx/source/docs/first_steps/examples/first_steps_9_widgets.py

```diff
@@ -0,0 +1,50 @@
+from bokeh.layouts import layout
+from bokeh.models import Div, RangeSlider, Spinner
+from bokeh.plotting import figure, output_file, show
+
+# prepare some data
+x = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+y = [4, 5, 5, 7, 2, 6, 4, 9, 1, 3]
+
+# set output to static HTML file
+output_file("first_steps.html")
+
+# create plot with circle glyphs
+p = figure(x_range=(1,9), plot_width=500, plot_height=250)
+points = p.circle(x=x, y=y, size=30, fill_color="#21a7df")
+
+# set up textarea (div)
+div = Div(text="""
+          <p>Select the circle's size using this control element:</p>
+          """,
+          width=200, height=30)
+
+# set up spinner
+spinner = Spinner(
+    title="Circle size",
+    low=0,
+    high=60,
+    step=5,
+    value=points.glyph.size,
+    width=200,
+    )
+spinner.js_link('value', points.glyph, 'size')
+
+# set up RangeSlider
+range_slider = RangeSlider(
+    start=0, end=10,
+    value=(p.x_range.start, p.x_range.end),
+    step=1, title="Adjust x-axis range"
+    )
```

Comment:
You are using three syntactic variants of the same multi-line constructor construct. Choose one, preferably this:
```py
name = Constructor(
    attr_0 = value_0,
    ...
    attr_n = value_n,
)
```
i.e. single indentation level, `)` on a new line flush with the first line, last attribute has a trailing comma. This way you can change variable name without shifting entire block and you can add attributes with the fewest number of changes (so producing nicer diffs). 

Having multiple attributes per line is fine, but group them by meaning, i.e. `step` and `title` don't go together.

Reply:
Thanks for pointing those out, I'll commit an update shortly! I have tried to be consistent with the examples in sphinx\source\docs\user_guide\examples, so I'd like to go with

```
range_slider = RangeSlider(
    title="Adjust x-axis range",
    start=0,
    end=10,
    step=1,
    value=(p.x_range.start, p.x_range.end),
)
```